test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/lang/CharacterTest.scala

```diff
@@ -434,9 +434,136 @@ class CharacterTest {
     assertEquals(0xff3b, Character.toLowerCase(0xff3b)) // ［ => ［
   }
 
-  @Test def toUpperCase_compare_char_and_codepoint(): Unit = {
+  @Test def toTitleCase_compare_char_and_codepoint(): Unit = {
     for (ch <- Character.MIN_VALUE to Character.MAX_VALUE)
-      assertEquals(Character.toUpperCase(ch), Character.toUpperCase(ch.toInt).toChar)
+      assertEquals(Character.toTitleCase(ch), Character.toTitleCase(ch.toInt).toChar)
+  }
+
+/*
+for (ch <- 0.toChar until Character.MAX_VALUE) {
+  val titleCase: Char = Character.toTitleCase(ch)
+  val upperCase: Char = Character.toUpperCase(ch)
+
+  if (titleCase != upperCase)
+    println(s"    assertEquals('$titleCase', Character.toTitleCase('$ch'))")
+}
+*/
+  @Test def toTitleCase_Char_UpperCase_Differences(): Unit = {
```

Comment:
I now realized that, since we have the exhaustive loop in `toTitleCase_compare_char_and_codepoint()`, we don't need to test anything else about `toTitleCase(Char)`. All the special cases should be tested in `toTitleCase(Int)` anyway, so this is redundant.

The rationale is: `toTitleCase_compare_char_and_codepoint()` can basically assume that `toTitleCase(Int)` is correct. If that one is correct, and we exhaustively test `toTitleCase(Char)` against `toTitleCase(Int)`, then `toTitleCase(Char)` is correct.

(and the test on the JVM will ensure that the *way* we test `totTitleCase(Char)` against `toTitleCase(Int)` is correct itself)